src/components/Projects.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div id="projects">
+    <h2 class="text-center text-3xl font-semibold text-indigo-100">
+      Our Projects
+    </h2>
+
+    <div
+      v-for="projects in ProjectData"
```

Comment:
I think we should make variable names consistent, here it's different than `projects`. Can you change to camel case?